qutebrowser/misc/backendproblem.py

```diff
@@ -414,6 +415,21 @@ def _check_software_rendering(self) -> None:
     def _assert_backend(self, backend: usertypes.Backend) -> None:
         assert objects.backend == backend, objects.backend
 
+    def _reenable_pyinstaller_macos_sandboxing(self) -> None:
+        """WORKAROUND for PyInstaller on macOS disabling sandboxing.
+
+        https://github.com/pyinstaller/pyinstaller/pull/6903
+        https://github.com/qutebrowser/qutebrowser/issues/7278
+        """
+        versions = version.qtwebengine_versions(avoid_init=True)
+        if not hasattr(sys, 'frozen') or \
+           not platform.system() == 'Darwin' or \
+           versions.webengine < utils.VersionNumber(6,):
```

Comment:
```suggestion
        if (
            not hasattr(sys, "frozen") or
            not utils.is_mac or
            versions.webengine < utils.VersionNumber(6)
        ):
```

- Using [implicit line continuation](https://www.pythonmorsels.com/breaking-long-lines-code-python/) instead of backslashes
- Using [`utils.is_mac`](https://github.com/qutebrowser/qutebrowser/blob/ce51fc522688cc8d644aaaea0b8cc6a8d81ddacc/qutebrowser/utils/utils.py#L67) which we already use elsewhere
- Dropping the unneeded `,` (no second argument anyways) for `VersionNumber`